assets/blocks/course-actions-block/course-actions/course-actions-edit.js

```diff
@@ -44,18 +108,34 @@ const CourseActionsEdit = ( { className, context: { postType } } ) => {
 		);
 	}
 
+	// Set class name for course status.
+	className = classnames( className, `is-status-${ courseStatus }` );
```

Comment:
Not a big deal here since `className` is a destructured variable, but I'd suggest creating a new variable to not modify the original variable coming from the function argument. Maybe `classNameWithStatus`? Or even add it directly to the `return` instead of creating a new variable.

Reply:
Changed in 0bdb9b0e8